rand_jitter/src/error.rs

```diff
@@ -56,15 +56,14 @@ impl ::std::error::Error for TimerError {
 }
 
 impl From<TimerError> for Error {
-    fn from(err: TimerError) -> Error {
+    fn from(_err: TimerError) -> Error {
         // Timer check is already quite permissive of failures so we don't
         // expect false-positive failures, i.e. any error is irrecoverable.
         #[cfg(feature = "std")] {
-            Error::with_cause(ErrorKind::Unavailable, "timer jitter failed basic quality tests", err)
+            Error::with_cause(ErrorKind::Unavailable, "timer jitter failed basic quality tests", _err)
         }
         #[cfg(not(feature = "std"))] {
```

Comment:
Maybe it's better to add `let _ = err;` here instead of renaming `err` to `_err`?

Reply:
Does it matter?

Reply:
I would prefer it, because I think it reflects intent better, but ultimately it's not important and I'm fine with either way.